src/xGadgetron/cGadgetron/include/sirf/Gadgetron/gadgetron_data_containers.h

```diff
@@ -840,6 +840,11 @@ namespace sirf {
 				(new AcquisitionsVector(acqs_info_, ignore_mask_));
 		}
 
+		virtual bool supports_array_view() const
+		{
+			return true;
```

Comment:
Is this really correct? I thought you needed an `ImageDataView` object? (haven't checked)

Reply:
no, this is a mistake - apparently I had `STIRAcquisitionDataInMemory` in mind

Reply:
no, it is a mistake, removed